app/react/V2/Components/Forms/DatePicker/DateRangePickerComponent.tsx

```diff
@@ -111,12 +130,12 @@ const DateRangePickerComponent = React.forwardRef(
                 </svg>
               </div>
               <InputField
+                id="from"
                 // @ts-ignore
```

Comment:
I've seen this ts-ignore in our InputField component and the input field tag in `V2/Components/Forms/DatePicker/DatePickerComponent.tsx`. 
Would this not by the attribute type="date" ? Or is this something specific to our implementation.

Reply:
That's right, this is because in our implementation we are using Flowbite-Datepicker, and the attributes in the input fields make the component behave like a calendar. Unfortunately, the clause datepicker is not recognized as a valid param of an HtmlInputElement